packages/manager-components/src/components/templates/onboarding/onboarding.component.tsx

```diff
@@ -15,7 +18,7 @@ export type OnboardingLayoutProps = PropsWithChildren<{
   title: string;
   orderButtonLabel: string;
   orderHref?: string;
-  description?: string;
+  description?: string | string[];
```

Comment:
Why not creating a new property to manage array. It could be confusing as `description` is expected to be a string